src/content/Concerts/02-03Fall.md

```diff
@@ -34,7 +34,7 @@ Section
 
 Overline = Program
 Work = Samuel Barber ~ Overture to *The School for Scandal*, op. 5
-Work = Leonard Bernstein ~ *Fancy Free*, op. 45
+Work = Leonard Bernstein ~ *Suite from Fancy Free*
```

Comment:
Wait ask me about this tonght. I might have changed it locally

Reply:
yogi